_issues/emails.md

```diff
@@ -0,0 +1,23 @@
+---  
+layout: post  
+title: "But isn't she a criminal for using a private email server?"  
+categories: controversy
+---  
+
+Source: [What the FBI Files Reveal About Hillary Clinton’s Email Server](http://www.politico.com/magazine/story/2016/09/hillary-clinton-emails-2016-server-state-department-fbi-214307)
+ 
+Q: Isn't Hillary is a criminal because she used a private email server?
+  
+A: Absolutely not. Hillary has admitted repeatedly that her use of a private email server was a mistake, and has apologized to the American people. There is no debate about that. However, an intensive FBI investigation showed that her actions were not criminal, and that she was not the first Secretary of State to use a private email account for business. Hillary used a personal email server merely for convenience, and not to circumvent security regulations, as her critics have claimed. Let's look at the facts:
+
+* Fact: The FBI investigation concluded that there was no criminal wrongdoing. As we all know, on July 5th, FBI Director James Comey took the unprecedented step of publicly criticizing Hillary for her use of private emails. However, he went on to say that the FBI investigation showed that there was insufficient evidence of any criminal wrongdoing.  
+
+* Fact: There were no requlations against using a private email server. During their investigation, FBI agents found there was “no restriction on use of personal email accounts for official business.”
+
+* Fact: Hillary did not conduct all of her business on her mobile device. Hillary had access to and used Sensitive Compartmented Information Facilities (SCIFs) throughout her tenure as Secretary of State. She even had SCIFs in her homes in Washington DC and Chappaqua, NY.
+
+* Fact: Only about a dozen people regularly emailed Hillary. Very few people had Hillary's email address. In fact, 68% of the email communications were between Hillary, and 3 top aides.
+
+* Fact: Hillary was "not an email person." Hillary strongly preferred in-person conversations and printed documents, often delivered via diplomatic pouch, over emails. Most of her communications were done outside of email.
```

Comment:
> "not an email person"

This makes her sound old-fashioned. Replace with "Very few of Hillary's communications were done over email" or something similar.